R/populateDaily.r

```diff
@@ -49,8 +49,6 @@ populateDaily <- function(rawData,qConvert,verbose = TRUE,interactive=NULL){  #
   dateFrame <- populateDateColumns(rawData$dateTime)
   localDaily <- cbind(localDaily, dateFrame[,-1])
   
-  localDaily$Date <- as.Date(localDaily$Date)
```

Comment:
I was under the impression you were only going to change the plotting functions, and if people used EGRET "as is" they wouldn't notice a change. Are you certain that deleting this line assures 99% of EGRET users will still get Date objects?